site/app/controllers/forum/ForumController.php

```diff
@@ -569,7 +569,7 @@ private function getSortedThreads($categories_ids, $max_thread, $show_deleted, $
     }
 
     public function getThreads(){
-        $pageNumber = !empty($_GET["page_number"]) && is_numeric($_GET["page_number"]) ? (int)$_GET["page_number"] : -1;
+        $pageNumber = !empty($_GET["page_number"]) && is_numeric($_GET["page_number"]) && $_GET["page_number"] >= 1 ? (int)$_GET["page_number"] : -1;
```

Comment:
Please test, if you input anything that is below 1 it will make the page number -1 (since ternary exp will be false), resulting in an exception when trying to run the offset query. A possible place to put this check might be where we try to load the threads.